packages/theme/modules/customer/pages/MyAccount/MyWishlist.vue

```diff
@@ -316,6 +320,29 @@ export default defineComponent({
             product,
             quantity,
           });
+          if (!cartError.value?.addItem) {
+            await removeItem({ product });
+
+            sendNotification({
+              id: Symbol('product_added_to_cart_from_wishlist_wishlist_page'),
+              message: app.i18n.t(`You added ${productGetters.getName(
+                product,
+              )} to your shopping cart.`) as string,
+              type: 'success',
+              icon: 'check',
+              persist: false,
+              title: 'Wishlist',
+            });
+          } else if (cartError.value.addItem.message) {
```

Comment:
```suggestion
          } else {
```

would that work? I don't think errors can have missing messages

Reply:
@sethidden I think the message should be there anyway, so this can be removed